### ISSUE TITLE
«When you sort by first order date in the admin, it freezes and breaks your admin account»

### ISSUE THREAD

**Issue by dmitrii-fediuk** [royalwholesalecandy/core#2]
>The task that we need assistance with is the first order date. 
We are unable to filter by this or sort.
When you sort by first order date in the admin, it freezes and breaks your admin account.

![2019-10-24_1602](https://user-images.githubusercontent.com/83204/68062428-93b8cf80-fd34-11e9-890a-d37f87bf0756.png)

![2019-10-24_1603](https://user-images.githubusercontent.com/83204/68062434-99161a00-fd34-11e9-9793-47133b7c8210.png)

[upwork.com/messages/rooms/room_ff8c0e6ae9e6cdfef5ce41dcd1dd08b5/story_a81990a9a32df4367696a597df2c5fdd](https://www.upwork.com/messages/rooms/room_ff8c0e6ae9e6cdfef5ce41dcd1dd08b5/story_a81990a9a32df4367696a597df2c5fdd)

**Comment by dmitrii-fediuk** [comment]
```
Column not found: 1054 Unknown column 'first_date' in 'order clause', query was: SELECT `main_table`.*, `pending_users`.`status` FROM `customer_grid_flat` AS `main_table`
 LEFT JOIN `itoris_pendingregistration_users` AS `pending_users` ON pending_users.customer_id = main_table.entity_id ORDER BY first_date ASC
 LIMIT 20
```

**Comment by dmitrii-fediuk** [comment]
Fixed in [1.0.1](https://github.com/royalwholesalecandy/core/releases/tag/1.0.1).
